modules/cnv_viewing_guest_ip_addresses.adoc

```diff
@@ -0,0 +1,59 @@
+[[viewing-guest-ip-addresses]]
+=== Viewing guest IP addresses
+
+You can view the IP addresses assigned to your Linux virtual machines by
+installing a QEMU guest agent on the host. When the guest agent is running,
+you can view the IP addresses per interface by checking the VMI status on
```

Comment:
I might s/IP addresses per interface/the virtual machine IP addresses for each interface

Reply:
Agreed. I wasn't really sure about "per" there.